packages/grafana-ui/src/components/Cascader/Cascader.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+// @ts-ignore
+import RCCascader from 'rc-cascader';
+import { BuildInPlacements as BuiltInPlacements } from 'rc-trigger';
```

Comment:
Not super wild about adding rc-trigger only to get this one type. It's yet another package that we need to manage(which we are kind of failing at atm). Can we redefine the type instead?

Reply:
rc-cascader already has rc-trigger as a dependency so adding it is really just to expose the `BuiltInPlacements` type

Reply:
looked into this a bit more and it seems like the version of rc-trigger that rc-cascader depends on doesn't have types, so you're right, I'll just have to redefine the type.